### ISSUE TITLE
Hi ,Can I use it in a production environment?

### ISSUE THREAD

**Issue by Zheby** [weldsorm/welds#20]

**Comment by lex148** [comment]
Hey Zheby. Great Question. This is a very loaded question, so let me break down my thoughts for you. 

When people ask this, They usually are asking one of the following. 

Q: Is welds bug free? 
A: Welds is fundamentally a wrapper around SQLX and Tiberius. For better or worse this means you get the stability/issues they provide :) .  Its main function is convenience for you. We have a very very strong push to test as much as we can. We are committed to addressing problems that occur as they come up.

Q: Is welds interface Stable (locked in, not changing)
A: Yes/No. We are about to release RC1 for the next version of welds. Currently the migration tests for MSSQL and MySql are the hold up. nothing wrong with them fundamentally, just going back to the first thought, everything needs to be tested to the extreme. There are very little changes in how you Interact with welds in the next version with the exception of Connections. This should be much simpler and pleasant to deal with. I recommend taking a look at the new examples if you would like to see what I mean here.  I would also recommend using the git version or soon to be RC1 version on crates.io if you are about to start a project. We are Days/Weeks out here . 

Q: Will Welds continue to be supported. 
A: Yes. I have no plans to stop supporting it. I am hopeful the community grows as well.

To sum up your question: We do....

**Comment by Zheby** [comment]
If possible I will read the code first and then join you
